storage-operator/pkg/apis/storage/v1alpha1/volume_types.go

```diff
@@ -23,9 +23,18 @@ type RawBlockDeviceVolumeSource struct {
 	DevicePath string `json:"devicePath"`
 }
 
+type LVMLVSource struct {
+	// Name of the LVM VolumeGroup on the node to create the LVM LogicalVolume to back
+	// the PersistentVolume
+	VgName string `json:"vgName"`
```

Comment:
Since we use `LVMLogicalVolume` etc elsewhere, this should be `VGName`, for consistency, no?

Reply:
Right :+1: